composer.json

```diff
@@ -10,7 +10,8 @@
         }
     ],
     "require": {
-        "php": ">=5.5.9"
+        "php": ">=5.5.9",
+        "laravel/installer": "^1.3"
```

Comment:
1.3 is back to 2015, so I doubt this will cause any trouble at all, but I'm curious--is there a specific reason you chose 1.3?

Thanks!